RedHatTraining/AD221-apps
# fix(route-files): Permissions issue in GE

SELinux blocks access to the mounted volume of the FTP container image.

This PR fixes the issue.

To test the PR:
* Test in the VM workstation
* You do not need to go through the main file, just run the python with `python3 ftp/run.py`,
*  Run the spring boot maven app from the solutions folder to check that the route reads files from the FTP server

The ftp run command should output something like this:
![image](https://user-images.githubusercontent.com/675470/155975817-3191af8b-c465-440a-9311-10c9e0d49044.png)
